### PR TITLE
Feature/add post method

### PR DESCRIPTION
**Pull request by neeraj3029**
Add a single-user-detail endpoint
```
GET
EP: /students/<id>/

Response
{
    "id": 3,
    "url": "http://127.0.0.1:8000/users/3/",
    "name": "Samagra1",
    "rollno": "1"
}
```
@akshi-11 
@samagragupta 
@rossoskull

### CODE REVIEW

File: api/students/views.py [hunk 1]
```diff
@@ -8,3 +8,19 @@
...
     serializer_class = UserSerializer
+
+class UserDetailView(generics.RetrieveUpdateDestroyAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+    def get(self, request, *args, **kwargs):
+        try:
+            required_user = self.queryset.get(pk=kwargs["pk"])
+            return Response(UserSerializer(required_user).data)
```
Comment (samagragupta) [inline api/students/views.py:19]: I think we should return `JSONResponse`.

Comment (samagragupta) [reply api/students/views.py:19]: ek baar check kr le jaha se tune kiya hai.

Comment (neeraj3029) [reply api/students/views.py:19]: khud se kia hai

Comment (neeraj3029) [reply api/students/views.py:19]: Screenshot 2019-10-27 at 12 58 35 PM
JSON reponse hai

Comment (neeraj3029) [reply api/students/views.py:19]: UserSerialiser se JSON object aa rha hai

**Review by samagragupta — APPROVED** [review]

**Review by akshitaagarwa11a — APPROVED** [review]

**Pull request merged**